packages/notte-browser/src/notte_browser/window.py

```
@@ -12,10 +12,10 @@
     TabsData,
     ViewportData,
 )
-from notte_core.common.config import FrozenConfig
+from notte_core.common.config import BrowserType, FrozenConfig, config
 from notte_core.errors.processing import SnapshotProcessingError
 from notte_core.utils.url import is_valid_url
-from notte_sdk.types import DEFAULT_VIEWPORT_HEIGHT, DEFAULT_VIEWPORT_WIDTH, BrowserType, Cookie, ProxySettings
+from notte_sdk.types import DEFAULT_HEADLESS_VIEWPORT_HEIGHT, DEFAULT_HEADLESS_VIEWPORT_WIDTH, Cookie, ProxySettings, SessionStartRequest
 from patchright.async_api import CDPSession, Locator, Page
 from patchright.async_api import TimeoutError as PlaywrightTimeoutError
 from pydantic import BaseModel, Field, model_validator
```

Ruff (0.11.9)

`pydantic.model_validator` imported but unused

Remove unused import: `pydantic.model_validator`

(F401)